404  and 500 page added

UmbrellaCrow612 opened the pull request:
Hi your website doesn't have a base 404 or 500 page so i added them

UmbrellaCrow612 committed:
404  and 500 page added

UmbrellaCrow612 committed:
Updated docs

Old command didn’t work

Commit:
chore: format file so it's the same as `_404.tsx`

UmbrellaCrow612 committed:
Docs

Added the right command

UmbrellaCrow612 committed:
Merge pull request #1 from synalice/UmbrellaCrow612/main

chore: format file so it's the same as `_404.tsx`

UmbrellaCrow612 committed:
Docs www

Removed bash

nikableh commented on www/README.md:
````diff
@@ -9,8 +9,7 @@ This is the fresh website source. The fresh website contains:
-deno run -A --watch main.ts
-```
+
+deno task start
````
You accidentally removed \``` ``` around the code block.

UmbrellaCrow612 replied:
Thanks

UmbrellaCrow612 committed:
Update README

Fixed the start command

UmbrellaCrow612 committed:
404 error page shows now

UmbrellaCrow612 committed:
toc spelling mistake fixed

UmbrellaCrow612 committed:
fixed spelling mistake for .MD file

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 46.44kb -> 29.06kb (37.43%)

/www/static/home-og.png -- 23.32kb -> 10.37kb (55.51%)
/tests/fixture/static/image.png -- 16.82kb -> 12.48kb (25.77%)
/www/static/illustration/lemon-squash.svg -- 3.23kb -> 3.13kb (3.09%)
/www/static/wordmark.svg -- 3.08kb -> 3.07kb (0.1%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

lucacasonato commented on docs/toc.json:
```diff
@@ -19,7 +19,7 @@
   "concepts": {
     "title": "Concepts",
     "pages": [
-      ["architechture", "Architecture"],
```
Please revert this. I'll tackle this later.

UmbrellaCrow612 committed:
Merge pull request #2 from UmbrellaCrow612/imgbot

[ImgBot] Optimize images

nikableh reviewed (commented):
Please don't change already existing files like that. They were fine as is.

lucacasonato reviewed (commented):
Please revert all of the changes that are not related to the 404 and 500 pages.

UmbrellaCrow612 committed:
Revert "[ImgBot] Optimize images"

UmbrellaCrow612 committed:
Revert "fixed spelling mistake for .MD file"

UmbrellaCrow612 committed:
Revert "toc spelling mistake fixed"

UmbrellaCrow612 committed:
Revert "Update README"

UmbrellaCrow612 committed:
Revert "Docs www"

UmbrellaCrow612 committed:
Revert "Updated docs"

UmbrellaCrow612 committed:
docs: removed the space

UmbrellaCrow612 committed:
docs: removed second space
